lib/Controller/DocumentController.php

```diff
@@ -294,6 +296,16 @@ public function remote(string $shareToken, string $remoteServer, string $remoteS
 		return new TemplateResponse('core', '403', [], 'guest');
 	}
 
+	/**
+	 * Open file on Source instance with token from Initiator instance
+	 */
+	#[PublicPage]
+	#[NoCSRFRequired]
+	public function remotePost(string $shareToken, string $remoteServer, string $remoteServerToken, ?string $filePath = null): TemplateResponse {
+		return $this->remote($shareToken, $remoteServer, $remoteServerToken, $filePath);
```

Comment:
This is needed as currently the routing cannot handle two routes with the same controller method name. We need one for POST and one for GET